apax/nn/builder.py

```diff
@@ -80,7 +80,7 @@ def build_descriptor(
     ):
         raise NotImplementedError("use a subclass to facilitate this")
 
-    def build_readout(self, head_config, is_feature_fn=False):
+    def build_readout(self, head_config, is_feature_fn=False, only_use_n_layers=None):
```

Comment:
I think `only_use_n_layers` is a bit difficult of a name. Maybe type hint `None|int` to show that you can select the layers and it is not of boolean type.